### ISSUE TITLE
Custom WP Editor

### ISSUE THREAD

**Issue by brucewrks** [wpsharks/wp-products#3]
I'd like to replace the default WordPress editor for Products with a couple of post meta boxes, and make this plugin generate product pages like we have at http://websharks-inc.com/.

Example: http://www.websharks-inc.com/product/wp-super-snow/

The post meta boxes would allow the site owner to set:
- A product image
- A list of buttons (URLs could be set through Shortcodes, to make it compatible with s2Member)
- Tabs with content like we have at WebSharks, Inc.

Additionally, it'd be cool if we could parse Markdown on the content for these tabs.

It shouldn't take too much effort to set up at all. The only issue would be making it backwards compatible with the standard WordPress content and excerpt functionality.

@jaswsinc Any thoughts on this?

**Comment by jaswrks** [comment]
That's a great idea. Go for it!

> It shouldn't take too much effort to set up at all. The only issue would be making it backwards compatible with the standard WordPress content and excerpt functionality.

You might consider leaving the editor and all of the default functionality as-is; this way a Product could actually be defined by the site owner as pretty much whatever they want to use it for. However, in the post editor you could add the post meta boxes, or even introduce a shortcode configurator there that would set a standard for what you have in mind, and then work to build more product-oriented functionality into that; i.e. shortcodes, meta boxes.

In short, what you define as a "Product" could still be optional.

**Comment by raamdev** [comment]
The WordPress editor box should be the product description and the post title box should be the product title that way you can pretty much leave the default custom post type options and then add several meta-boxes that allow you to add other information about the product.

**Comment by raamdev** [comment]
i.e., what @jaswsinc said. :)
